public/js/loginValidation.js

```diff
@@ -1,47 +1,47 @@
-function isValidEmail(elementID) {
-    let inputVal = document.getElementById(elementID);
+let isValidEmail = (elementID) => {
+    let inputVal = document.getElementById(elementID)
+    let userEmailWarning = document.getElementById('userEmailWarning')
     let email = inputVal.value.trim();
-    let re = new RegExp("^\\w+([\\.-]?\\w+)*@\\w+([\\.-]?\\w+)*(\\.\\w{2,3})+$");
-
+    let re = new RegExp("^\\w+([\\.-]?\\w+)*@\\w+([\\.-]?\\w+)*(\\.\\w{2,3})+$")
```

Comment:
could we rename variable 're' something a little more semantic?

Reply:
done